### ISSUE TITLE
Incorrect icons size in the left panel

### ISSUE THREAD

**Issue by RussKie** [gitextensions/gitextensions#9964]
### Environment

- Build 47754d7276d2b137c72b5bd480be0d491a037844 (Dirty)

### Issue description

@h0lg before #9723 the left panel's context menu icons looked liked this:
![image](https://user-images.githubusercontent.com/4403806/163712240-4a2dcbcd-6f4a-41da-bebb-76712e4726ef.png)

Now the icons look like this:
![image](https://user-images.githubusercontent.com/4403806/163712251-5b97667e-6bee-4496-9524-95df410deba1.png)

### Steps to reproduce

Open the context menu in the left panel.

**Comment by h0lg** [comment]
The icons all seem a bit bigger than they were before - looking at your screenshots side by side. If that's what you mean, I'll see what I can do about that.

**Comment by RussKie** [comment]
There's a helper method that scales icons - `DpiUtil.Scale` (e.g., `button.Image = DpiUtil.Scale(button.Image);`). I guess that's what missing.

**Comment by h0lg** [comment]
> There's a helper method that scales icons

Thanks for the heads up!

I looked a the changes in `/GitUI/BranchTreePanel/RepoObjectsTree.Designer.cs` and found that the designer (without my doing) inserted the following line 

`this.menuMain.ImageScalingSize = new System.Drawing.Size(20, 20);`

whereas most of the icons are 16 by 16 px. Changing or removing that looks like an easy way to change the size of all icons at once.

**Comment by h0lg** [comment]
@RussKie I've created PR #9966 for the bug fix.

**Comment by gerhardol** [comment]
See also #7942

**Comment by h0lg** [comment]
@gerhardol I've looked at #7942 and don't understand how it relates to this issue. Please explain?

**Comment by pmiossec** [comment]
@h0lg I don't think it is related to your problem because you have a sizing problem with already used icons that were working fine.

I think that's just a heads up on a strange behavior when having sizing problem when using an image...

**Comment by gerhardol** [comment]
[pmiossec](https://github.com/pmiossec) is right, I did not see if any icons were changed.
